java/org/contikios/cooja/Cooja.java

```diff
@@ -1706,11 +1706,11 @@ public Boolean work() {
   }
 
   /**
-   * Same as the {@link #startPlugin(Class, Cooja, Simulation, Mote)} method,
+   * Same as the {@link #tryStartPlugin(Class, Cooja, Simulation, Mote)} method,
```

Comment:
I think this should be `startPlugin(Class, Cooja, Simulation, Mote, activate)`.

Reply:
You're right, fixed.